app/views/wikies/_form.html.erb

```diff
@@ -11,6 +11,14 @@
     </div>
   <% end %>
 
+  <% if current_user.admin? || (current_user.premium? && @wiky.user_id == current_user.id) %>
```

Comment:
The current line 14 is a little long and unclear as to what the intention is. If we could wrap this expression up into a single method with a descriptive name then it'll make this easier to understand the next time we read this code.

We can add a method to `WikiPolicy` that describes the intent of this expression and so make this line more readable.

For example, in `app/views/wikies/_form.html.erb:
```ruby
<% if policy(@wiky).privatize? %>
```

And in `WikiPolicy`:

```ruby
  def privatize?
    wiki = record
    user.present? && ( user.admin? || (user.premium && wiki.user == user) )
  end
```